### PR TITLE
merging jupyterhub and binder teams

### LINKED ISSUE #269: Merge the Binder and JupyterHub teams
Following a discussion in our community meeting:

right now, we have two different teams listed on the teams page:

https://jupyterhub-team-compass.readthedocs.io/en/latest/team.html

A jupyterhub team, and a binder team.

These two teams are pretty well overlapping with each other, and also have many of the same responsibilities, projects, etc.

So: **what if we merged these into a single team, and allowed team members to note if they'd like to emphasize a particular part of the project on which they work? (e.g., Binder)**. Then we'd have a single team, and we'd use the same "color" roles for it.

This would also resolve #266

### PR DESCRIPTION
**Pull request by choldgraf**
A first pass at merging the jupyterhub/binder teams in the docs. This doesn't change any team membership or documentation around it. Anybody that was in the JupyterHub team but not on the Binder team had "blue team" added to their entry. Also note that while I included information about team focus in the YAML file, I don't put it into the HTML yet just to try and keep this PR simpler.

What do folks think?

closes #269

### CODE REVIEW

File: docs/team.rst [hunk 1]
```diff
@@ -29,27 +29,24 @@ to Binder, JupyterHub, and Project Jupyter as a whole.
...
+.. TODO: Update the governance page so that it isn't binder-specific
 
-Binder Team
------------
+.. TODO: Include Binder / JupyterHub team focus in the HTML somehow
 
-Binder's governance and team structure is defined in the
-:ref:`binder-governance` page. Below we list the current team members
-of Binder.
+Here is the current JupyterHub/Binder team:
```
Comment (consideRatio) [inline docs/team.rst:43]:
```suggestion
Here is the current JupyterHub team:
```

Right?

File: docs/team.rst [hunk 2]
```diff
@@ -29,27 +29,24 @@ to Binder, JupyterHub, and Project Jupyter as a whole.
...
+overlapping team membership. Both are part of `Project Jupyter <http://jupyter.org/>`_
+and are developed by an open community of contributors.
 
-(listed alphabetically, with affiliation, and main areas of contribution)
-
-Min Ragan-Kelley acts as the team lead for the JupyterHub organization.
-
-.. include:: team/contributors-jupyterhub.txt
+The process and governance for both Binder and JupyterHub can be found in the
+:ref:`jupyterhub-governance` :ref:`binder-governance` pages.
```
Comment (betatim) [inline docs/team.rst:37]:
```suggestion
:ref:`jupyterhub-governance` :and ref:`binder-governance` pages.
```

**Comment by consideRatio** [comment]
LGTM!

**Comment by betatim** [comment]
It is done!

**Pull request merged**